webapp/lib/controller.dart

```diff
@@ -475,8 +474,7 @@ void initUI() {
         ..addAll(added)
         ..addAll(modified);
       view.conversationListSelectView.updateConversationLists(shards);
-    }
-  );
+    }, showAndLogError);
```

Comment:
in testing I saw that the load spinner still appears, which might give the impression to the user that something is still loading. I think something like this will do the trick to hide it:

```suggestion
    }, (error, stacktrace) {
      view.conversationListPanelView.hideLoadSpinner();
      showAndLogError(error, stacktrace);
    });
```

Reply:
Fixed